common/lib/xmodule/xmodule/templates/problem/imageresponse.yaml

```diff
@@ -4,19 +4,29 @@ metadata:
     markdown: !!null
 data: |
   <problem>
-    <p>
-      An image mapped input problem presents an image for the student.
-      Input is given by the location of mouse clicks on the image.
-      Correctness of input can be evaluated based on expected dimensions of a rectangle.
-    </p>
-    <p>Which animal shown below is a kitten?</p>
-    <imageresponse>
-      <imageinput src="https://studio.edx.org/c4x/edX/DemoX/asset/Dog-and-Cat.jpg" width="640" height="400" rectangle="(385,98)-(600,337)" alt="Two animals side by side. On the left, a brown furry animal with floppy ears and a large dark nose, on the right, a smaller furry animal with pointy ears and a small pink nose."/>
-    </imageresponse>
-    <solution>
-      <div class="detailed-solution">
-        <p>Explanation</p>
-        <p>The animal on the right is a kitten. The animal on the left is a puppy, not a kitten.</p>
-      </div>
-    </solution>
+  <p>
+  In an image mapped input problem, also known as a "pointing on a picture"
+  problem, students click inside a defined region in an image. You define this
+  region by including coordinates in the body of the problem. You can define
+  one rectangular region, multiple rectangular regions, or one non-rectangular
+  region. For more information, see "Image Mapped Input Problem" in
+  <i>Building and Running an edx Course</i>.
+  </p>
+  <p>You can use the following example problem as a model.</p>
+  <p>_____________________________________________________________________________</p>
+  <p>What country is home to the Pyramids as well as the cities of Cairo and
```

Comment:
I *think* that you capitalize the Great Pyramid of Giza, but when you refer to all of them they are the lower-case pyramids of Giza.